datumaro/plugins/voc_format/format.py

```diff
@@ -124,181 +129,278 @@ class VocPath:
     }
 
 
-LabelMapConfig = Dict[str, Tuple[Optional[RgbColor], List[str], List[str]]]
-"""A type representing a label map config"""
-# Not totally type-correct, tuple elements are supposed to support modification.
-# Therefore, the tuple is typically a list
-# TODO: refactor, make type annotations conform with actual usage
+@define
+class VocLabelInfo:
+    color: Optional[RgbColor] = None
+    parts: Set[str] = field(factory=set, converter=set)
+    actions: Set[str] = field(factory=set, converter=set)
 
 
-def make_voc_label_map() -> LabelMapConfig:
-    labels = sorted(VocLabel, key=lambda l: l.value)
-    label_map = OrderedDict((label.name, [VocColormap[label.value], [], []]) for label in labels)
-    label_map[VocLabel.person.name][1] = [p.name for p in VocBodyPart]
-    label_map[VocLabel.person.name][2] = [a.name for a in VocAction]
-    return label_map
+if sys.version_info.minor < 9:
```

Comment:
```suggestion
if sys.version_info < (3, 9):
```

Otherwise, this is liable to break if Python 4 is ever released.